.travis.yml

```diff
@@ -18,6 +18,10 @@ jobs:
       name: YAML lint
     - script: ./hack/shellcheck.sh
       name: shellcheck
+    - script: ./hack/tf-fmt.sh
+      name: Terraform format
+    - script: ./hack/gotest.sh
+      name: go test
```

Comment:
Shouldn't these replace the "Terraform tests" and "Installer unit tests" entries below?

Reply:
yes they should, and now they do, thanks